policies/aws.route53.-.connectiontime.json

```diff
@@ -0,0 +1,27 @@
+{
+  "policy": {
+    "actions": [
+      {
+        "category": 2,
+        "type": "event"
+      }
+    ],
+    "conditions": [
+      {
+        "analytic": "baselineDeviation",
+        "operator": ">",
+        "wildcard": "aws.route53.connectiontime.*"
+      }
+    ],
+    "deleted": false,
+    "description": "ConnectionTime is higher than usual. ConnectionTime is the average time, in milliseconds, that it took Route 53 health checkers to establish a TCP connection with the endpoint. You can view ConnectionTime for a health check either across all regions or for a selected geographic region.",
```

Comment:
Can you view ConnectionTime for a health check across all regions? I don't see a metric for that. What should I be looking for?

Reply:
This metric depends on AWS settings and available not for all elements. It shows up when Latency Graphs are enabled in AWS for a particular check. I enabled them in AWS for tstLatencyGraphs1 element: http://take.ms/amRcJ

Reply:
Pardon me, it looks like I misunderstood your question. I took this description from Amazon website. Possibly it is related to the AWS console, as it shows a dropdown menu to select Global or a region: http://take.ms/DFbRy But, in the policy I track all possible regions by regex: "wildcard": "aws.route53.connectiontime.*"
Should I change the description?